.github/workflows/linting.yml

```
@@ -1,49 +1,57 @@
-
-    - name: Run ruff (formatter)
-      run: ruff format --check
-
-    - name: Run mypy
-      run: mypy .
\ No newline at end of file
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)